server/api/documents.js

```diff
@@ -366,11 +366,7 @@ router.post("documents.drafts", auth(), pagination(), async ctx => {
   };
 });
 
-router.post("documents.info", auth({ required: false }), async ctx => {
-  const { id, shareId } = ctx.body;
-  ctx.assertPresent(id || shareId, "id or shareId is required");
-
-  const user = ctx.state.user;
+async function loadDocument({ id, shareId, user }) {
```

Comment:
Because the document load and auth logic is the same for `documents.info` and `documents.export` we pull it out into it's own function to reduce duplication.